treeherder/model/models.py

```diff
@@ -39,9 +39,7 @@
 @python_2_unicode_compatible
 class NamedModel(models.Model):
     id = models.AutoField(primary_key=True)
-    name = models.CharField(max_length=50, db_index=True)
-    description = models.TextField(blank=True)
```

Comment:
Since we're removing description here, we'll need to add it back for `RepositoryGroup`.